### ISSUE TITLE
[Headless] [Windows] Terminating server does not save the game

### ISSUE THREAD

**Issue by Sweet-Tan** [OpenRCT2/OpenRCT2#4860]
**OS:** Windows 7 Pro x64
**Version:** 0.0.5
**Commit/Build:**  7ac7c1e , 51fcfe3

When terminating a headless server with ^C , game is not saved.
Command is normally run within a Batch file, but the issue occurs regardless.
The OpenRCT install is located on a different letter drive than the save and user data location. Auto-saves do work correctly.

The OpenRCT Servers affected run in the same Windows 7 VM on top of XenServer 6.5. They utilize the following arguments: --port --headless --console --verbose --user-data-path

- [ ] Reproducible in RCT2 (vanilla)?
- [x] Multiplayer?

**Steps to reproduce:**
1. Run game with command similar to "C:\Program Files\OpenRCT2\openrct2.exe" host "E:\testserver\testserver.sv6" --port 11753 --headless --console --verbose --user-data-path E:\testserver
2. Terminate console with Ctrl+C

**Comment by AaronVanGeffen** [comment]
Personally, I don't think it's desirable to be saving the game state when terminating the application — especially not though a `SIGKILL`.

Maybe we could introduce saving though a different signal, though — say, `SIGUP`? I'm not sure what the Windows equivalent of that is, though.

**Comment by Sweet-Tan** [comment]
Good point. I agree `SIGKILL` shouldn't save the game. If that's the case, we need a way of saving the game directly from the console. I'm aware we cannot run commands directly from the console, but it may be worth bringing up #2854
